app/controllers/api/v1/customers_controller.rb

```diff
@@ -70,6 +70,24 @@ def destroy
         end
       end
 
+      def checkout_url
+        customer = current_organization.customers.find_by(external_id: params[:customer_external_id])
+
+        result = ::Customers::GenerateCheckoutUrlService.call(customer:)
+
+        if result.success?
+          render(
```

Comment:
What about using the `V1::PaymentProviders::CustomerCheckoutSerializer`?
It's used in the webhook for the same purpose